readthedocs/oauth/migrate.py

```diff
@@ -296,6 +296,16 @@ def get_migration_targets(user) -> list[MigrationTarget]:
     return targets
 
 
+def has_projects_pending_migration(user) -> bool:
+    """
+    Check if the user has any projects pending migration to the GitHub App.
+
+    This includes all projects that are connected to the old GitHub OAuth App,
+    where the user has admin permissions and the GitHub App is installed.
+    """
+    return any(_get_projects_missing_migration(user))
```

Comment:
It seems that `_get_projects_missing_missing_migration` and `get_valid_projects_missing_migration` are the same -- in particular based on its docstring.

However, one returns _all the projects_ and the other only returns the projects that `has_installation and is_admin`. Can we update the docstring to reflect this?

There is some confusion because we are talking about "admin" permission on the `Project` object but also "admin" permission on the GitHub repository. We should clarify what we refer to on each case.